corehq/apps/export/views/download.py

```diff
@@ -214,7 +214,7 @@ def page_context(self):
             'show_date_range': self.show_date_range,
             'check_for_multimedia': self.check_for_multimedia,
             'sms_export': self.sms_export,
-            'user_types': HQUserType.HqUser_all_friendly_names()
+            'user_types': HQUserType.get_all_friendly_names()
```

Comment:
phew, I was worried you were gonna keep the `HqUser_` prefix for all of these, makes sense to distinguish while doing the refactor, though.